moped-editor/src/views/dashboard/DashboardView.js

```diff
@@ -199,7 +201,12 @@ const DashboardView = () => {
       field: "current_phase",
       editable: "never",
       render: entry =>
-        buildStatusBadge(entry.current_phase, entry.status_id),
+        buildStatusBadge(
```

Comment:
Does this need to be wrapped in a function that only returns the component, or could the component go here instead of the function?

Reply:
You're right, thanks for pointing that out! That's much cleaner. I refactored this and will update before merging.